GDJS/Runtime/howler-sound-manager/howler-sound-manager.js

```diff
@@ -14,24 +14,28 @@
  * @memberof gdjs
  * @class HowlerSound
  */
-gdjs.HowlerSound = function (o) {
-  Howl.call(this, o);
+gdjs.HowlerSound = function (howl) {
+  this._howl = howl;
+  // Get a unique sound playing ID.
+  this.id = howl.play();
```

Comment:
Is getting this id really necessary? Seems like we don't need it because we only have one sound managed in a HowlerSound?
I'm also concerned of launching the play and immediately stopping it. Not sure how various browsers will react to doing this for a large number of sounds.

Reply:
The id is necessary as one file could run two times in parallel. Though I agree I did it wrong, I probably should keep the id undefined until play is called.